tests/helpers.py

```diff
@@ -65,24 +65,36 @@ def create_resource(
     return ocs_obj
 
 
-def create_pod(desired_status=constants.STATUS_RUNNING, wait=True, **kwargs):
+def create_pod(interface_type=None, pvc=None, desired_status=constants.STATUS_RUNNING, wait=True):
```

Comment:
Did you check that no one using this func?

Reply:
Right, fixed